### PR TITLE
Add ReadOnly field supporting writable deploy keys.

### PR DESCRIPTION
**Pull request by vtemian**
Related to https://github.com/go-gitea/gitea/pull/3225

### CODE REVIEW

File: gitea/repo_key.go [hunk 1]
```diff
@@ -13,10 +13,10 @@ import (
 
 // DeployKey a deploy key
 type DeployKey struct {
-	ID       int64     `json:"id"`
-	Key      string    `json:"key"`
-	URL      string    `json:"url"`
-	Title    string    `json:"title"`
+	ID    int64  `json:"id"`
```
Comment (lafriks) [inline gitea/repo_key.go]: This change does not seems to be related to this PR as only whitespace are changed here

**Review by lafriks — CHANGES_REQUESTED** [review]

File: gitea/repo_key.go (hunk 1 above)
Comment (vtemian) [reply gitea/repo_key.go]: Yeah, I just run `fmt` and I committed the changes.
Those changes need to be done in another PR?

Comment (lafriks) [reply gitea/repo_key.go]: I think yes

**Review by lafriks — APPROVED** [review]

**Comment by lafriks** [comment]
LGTM

**Comment by lunny** [comment]
LGTM

**Pull request merged**